Cargo.toml

```diff
@@ -123,6 +123,8 @@ object = { opt-level = 3 }
 rustc-demangle = { opt-level = 3 }
 
 [profile.release]
+lto = "thin"
```

Comment:
Mind adding a comment here noting we observed ~no increase in compile time but a ~10% increase in runtime performance?